app/jobs/expire_ephemeral_state_job.rb

```diff
@@ -2,6 +2,6 @@ class ExpireEphemeralStateJob < ApplicationJob
   queue_as :default
 
   def perform
-    EphemeralState.where("created_at < ?", 60.minutes.ago).delete_all
+    EphemeralState.where("created_at < ?", 2.hours.ago).delete_all
```

Comment:
In these expire jobs, can we somehow access the session store's `expire_after` parameter?  Because these are always going to be the same.

Reply:
mmm that's a nice idea, let me take a look